index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Linkbrary</title>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="./src/reset.css" />
+    <link rel="stylesheet" href="./src/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="index.html">
+          <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+        </a>
+        <a class="cta cta-short" href="signin.html">
+          <span>로그인</span>
+        </a>
+      </nav>
+      <div class="hero-header">
+        <h1 class="slogan">
+          <span class="slogan-gradient background-clip-text"> 세상의 모든 정보</span>를
+          <br />
+          쉽게 저장하고 관리해 보세요
+        </h1>
+        <a class="cta cta-long" href="signup.html">
+          <span>링크 추가하기</span>
+        </a>
+        <img src="./images/hero.png" class="hero-image" alt="Linkbrary 서비스 소개" />
+      </div>
+    </header>
+    <article>
+      <section>
+        <h2 class="title">
+          <span class="title-1-gradient background-clip-text"> 원하는 링크</span>를
+          <br />
+          저장하세요
+        </h2>
+        <p class="description">
+          나중에 읽고 싶은 글, 다시 보고 싶은 영상,
+          <br />
+          사고 싶은 옷, 기억하고 싶은 모든 것을
+          <br />
+          한 공간에 저장하세요.
+        </p>
+        <img src="./images/image1.png" class="content-image" alt="링크의 내용이 담긴 카드들" />
+      </section>
+      <section>
+        <h2 class="title">
+          링크를 폴더로
+          <br />
+          <span class="title-2-gradient background-clip-text">관리</span>하세요
+        </h2>
+        <p class="description">
+          나만의 폴더를 무제한으로 만들고
+          <br />
+          다양하게 활용할 수 있습니다.
+        </p>
+        <img src="./images/image2.png" class="content-image" alt="폴더 이름 변경 기능" />
+      </section>
+      <section>
+        <h2 class="title">
+          저장한 링크를
+          <br />
+          <span class="title-3-gradient background-clip-text">공유</span>해 보세요
+        </h2>
+        <p class="description">
+          여러 링크를 폴더에 담고 공유할 수 있습니다.
+          <br />
+          가족, 친구, 동료들에게 쉽고 빠르게 링크를
+          <br />
+          공유해 보세요.
+        </p>
+        <img src="./images/image3.png" class="content-image" alt="폴더 공유 기능" />
+      </section>
+      <section>
+        <h2 class="title">
+          저장한 링크를
+          <br />
+          <span class="title-4-gradient background-clip-text">검색</span>해 보세요
+        </h2>
+        <p class="description">중요한 정보들을 검색으로 쉽게 찾아보세요.</p>
+        <img src="./images/image4.png" class="content-image" alt="링크 검색 기능" />
+      </section>
+    </article>
+    <footer>
+      <div class="footer-box">
+        <span class="copyright">©codeit - 2023</span>
```

Comment:
요런 방식도 있으니 참고해주세요
https://developer.mozilla.org/en-US/docs/Glossary/Entity

```suggestion
        <span class="copyright">&copy;codeit - 2023</span>
```